invenio_search/api.py

```diff
@@ -54,7 +54,8 @@ def query(self):
             tree = tree.accept(walker)
         return tree
 
-    def search(self, user_info=None, collection=None, **kwargs):
+    def search(self, user_info=None, collection=None, index="records*",
```

Comment:
We were wondering how would the client code look like? If say, we want to make a search in the `Jobs` index ?

Can you provide an example? Thanks